ui/app/src/Views/AlertList/Views.elm

```diff
@@ -91,6 +91,12 @@ alertGroups activeId filter groupBar alerts =
                         (alertList activeId labels filter)
                         (Dict.get labels grouped)
                 )
+            |> (\list ->
+                    if List.isEmpty list then
+                        [ Utils.Views.error "No alerts found" ]
```

Comment:
Yeah, we had this message in different place and it was never triggered. Now it is